Sources/MapboxNavigation/UserHaloCourseView.swift

```diff
@@ -61,6 +61,24 @@ open class UserHaloCourseView: UIView, CourseUpdatable {
         haloView.backgroundColor = .clear
         addSubview(haloView)
     }
+    
+    /**
+     Transforms the location of the user halo course view.
+     */
+    public func update(location: CLLocation, pitch: CGFloat, direction: CLLocationDegrees, animated: Bool, navigationCameraState: NavigationCameraState) {
```

Comment:
Ah, it was probably me, who removed this part some time ago. At the same time I think in default case to omit code duplication we can use something like this in `CourseUpdatable` protocol extension:
```swift
if !(self is UserHaloCourseView) {
    let angle = CGFloat(CLLocationDegrees(direction - location.course).toRadians())
    self.layer.setAffineTransform(CGAffineTransform.identity.rotated(by: -angle))
}
```